craco.config.js

```diff
@@ -0,0 +1,6 @@
+// craco.config.js
```

Comment:
What is CRACO? I never heard of it. I cannot approve this without knowing some stuff...

Reply:
https://www.npmjs.com/package/@craco/craco

Reply:
✅